actors/miner/src/monies.rs

```diff
@@ -267,10 +271,32 @@ pub fn initial_pledge_for_power(
     let lock_target_denom = LOCK_TARGET_FACTOR_DENOM;
     let pledge_share_num = qa_power;
     let network_qa_power = network_qa_power_estimate.estimate();
-    let pledge_share_denom = cmp::max(cmp::max(&network_qa_power, baseline_power), qa_power);
+
+    // Once FIP-0081 has fully activated, additional pledge will be 70% baseline
+    // pledge + 30% simple pledge.
+    const FIP_0081_ACTIVATION_PERMILLE: i64 = 300;
+    // gamma = 1000 - 300 * (epochs_since_ramp_start / ramp_duration_epochs).max(0).min(1)
+    let skew = ((epochs_since_ramp_start * FIP_0081_ACTIVATION_PERMILLE)
+        / ramp_duration_epochs.max(1) as i64)
+        .max(0)
```

Comment:
The uses of max and min are clever, but maybe too much so? I'm finding it too hard to mentally verify the expected behaviour around cases like negative epochs_since_start and zero ramp duration. Tests will help with confidence here, but also an if/match construction could take out some of those edge cases clearly.

Reply:
Done.

Reply:
This way is definitely easier to follow thanks